src/connector/src/with_options.rs

```diff
@@ -201,6 +201,10 @@ pub trait WithPropertiesExt: Get + GetKeyIter + Sized {
             })
             .unwrap_or(false)
     }
+
+    fn is_refreshable_connector(&self) -> bool {
+        false
+    }
```

Comment:
This decides whether a table can be refreshed. So not possible to create refreshable table in this PR